addons/im_support/static/src/js/mail_manager.js

```diff
@@ -5,8 +5,8 @@ var MailManager = require('mail.Manager');
 
 var core = require('web.core');
 var session = require('web.session');
-var WebClient = require('web.WebClient');
 
+require('im_support.SupportBus');
```

Comment:
For info im_support is disabled on our prod, AL decided yesterday to delete the whole addon, which I will do this week then.

Reply:
sadness ^^